### PR TITLE
updated to vite

### PR DESCRIPTION
**Pull request by PabloGancharov**
Screenshot 2024-01-25 at 11 11 11

### CODE REVIEW

File: package-lock.json
Comment (giulyferto) [inline package-lock.json:1]: is it okay to push changes on this file?

File: src/App.css
Comment (giulyferto) [inline src/App.css:1]: If this file is empty we can delete it

File: src/index.css
Comment (giulyferto) [inline src/index.css:1]: Same here

File: lib/components/ForgeViewer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            }
+            const onDocumentLoadFailure = (code: any, message: any, errors: any) => {
+                console.error({ code, message, errors });
+            }
+            viewer.setLightPreset(0);
+            console.log(viewer)
+            // viewer.resize();
+            // window.addEventListener('resize', function () {
+            //     viewer.resize();
+            // });
```
Comment (FedericoFernandezAguiar) [inline lib/components/ForgeViewer/index.tsx:61]: Is all these necessary?

File: package-lock.json
Comment (PabloGancharov) [reply package-lock.json:1]: this time is OK, because I delete everything and related all again using vite

**Review by RominaDosSantos — APPROVED** [review]

File: lib/components/ForgeViewer/index.tsx (hunk 1 above)
Comment (PabloGancharov) [reply lib/components/ForgeViewer/index.tsx:61]: nop

File: src/index.css
Comment (PabloGancharov) [reply src/index.css:1]: I keep it as a placeholder

**Pull request merged**